src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubTriggerAttribute.cs

```diff
@@ -35,7 +35,6 @@ public EventHubTriggerAttribute(string eventHubName)
         /// <summary>
         /// Gets or sets the optional app setting name that contains the Event Hub connection string. If missing, tries to use a registered event hub receiver.
         /// </summary>
-        [AppSetting]
```

Comment:
Triggers do not go through the AttributeCloner as they do not use binding rules. This setting wasn't doing anything, so removing to reduce confusion.